src/plugin/regions/regions.js

```diff
@@ -0,0 +1,412 @@
+/**
```

Comment:
can you add a since tag to this file (I would use version 3.4.0): https://jsdoc.app/tags-since.html

Reply:
also in `region.js`?

Reply:
oh sorry, I meant region.js. Perhaps in regions.js you can add a note that it was moved etc in 3.4.0. I'm also not sure if it should be called `src/regions/regions.js` or `src/regions/index.js`. I don't like the word `regions` used twice there.

Reply:
We can call it or `index.js` or `regionsPlugin.js`, what you prefer?

Reply:
not sure, what about you @katspaugh ?

Reply:
there are also merge conflicts in `src/plugin/regions.js`. Maybe you should reset the master branch of your fork @marizuccara, there are a lot of unrelated changes in your pull requests.

Reply:
`index.js` does sound better. You can import it with `import regions from 'plugin/regions` then.

Reply:
agreed

Reply:
But naming it index.js, when creating the bundle of regions plugin, it will be `wavesurfer.index.js` and `wavesurfer.region.js`, instead of  `wavesurfer.regions.js` and `wavesurfer.region.js`. Correct?

Reply:
Yea, the way it's setup now, it will. So the build script needs to include a ``src/plugin/*/index.js` as well and use it's parent directory name for the plugin.

We could also refactor all plugins afterwards and move them into a directory and use the `index.js` convention but that's for a different pull request.

Reply:
So, in this way, only `wavesurfer.regions.js` have to be created, and not also `wavesurfer.region.js`?

Reply:
Yes.